src/main/java/org/folio/rmapi/model/PackageData.java

```diff
@@ -0,0 +1,108 @@
+package org.folio.rmapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PackageData {
+  @JsonProperty("packageId")
+  private Integer packageId;
+  @JsonProperty("packageName")
+  private String packageName;
+  @JsonProperty("vendorId")
+  private Integer vendorId;
+  @JsonProperty("vendorName")
+  private String vendorName;
+  @JsonProperty("isCustom")
+  private Boolean isCustom;
+  @JsonProperty("titleCount")
+  private Integer titleCount;
+  @JsonProperty("isSelected")
+  private Boolean isSelected;
+  @JsonProperty("selectedCount")
+  private Integer selectedCount;
+  @JsonProperty("contentType")
+  private String contentType;
+  @JsonProperty("visibilityData")
+  private VisibilityInfo visibilityData;
+  @JsonProperty("customCoverage")
+  private CoverageDates customCoverage;
+  @JsonProperty("isTokenNeeded")
+  private Boolean isTokenNeeded;
+  @JsonProperty("allowEbscoToAddTitles")
+  private Boolean allowEbscoToAddTitles;
+  @JsonProperty("packageType")
+  private String packageType;
+  @JsonProperty("proxy")
+  private Proxy proxy;
+  @JsonProperty("packageToken")
+  private VendorToken packageToken;
```

Comment:
Are we using class VendorToken here because structure of both vendorToken and packageToken is the same or is it a typo?

Reply:
Yes, in RM API docs it is the same object with same structure, still it seems strange to have this class named VendorToken, renamed it to TokenInfo